### ISSUE TITLE
Legend - dark colors against grey

### ISSUE THREAD

**Issue by dcorliss** [planetfederal/sfpark.org#22]
In the legends, it's hard to tell the darkest color (green or blue) from the grey color in the little circles. Is there another way to show this information?

**Comment by bartvde** [comment]
This one needs to be discussed with Rolando.

**Comment by scrollie** [comment]
I agree that there is not a lot of contrast between the darkest swatches and the pop-up background—but I also think this issue depends on the color settings of your own display (my two monitors show up ok, though not optimal). Our options here are to either change the background, change the colors, or trust that the context of the legend bar and the markers on the map are enough to make clear what's going on. What do y'all think?
